static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -232,6 +257,98 @@ const errorHelpTypeResourceMap: Record<
       linksByPlatform: {},
     },
   },
+  [ErrorHelpType.NEXTJS_ROUTER_NOT_MOUNTED]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Next.js applications when the useRouter hook is used incorrectly. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>NextRouter not mounted errors</b>}
+      ),
+      links: [
+        {
+          text: t('Fixing "NextRouter was not mounted" errors in Next.js'),
+          link: 'https://sentry.io/answers/error-nextrouter-was-not-mounted/',
+        },
+      ],
+      linksByPlatform: {},
+    },
+  },
+  [ErrorHelpType.UNBOUND_LOCAL_ERROR]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Python applications when a variable is defined in both global and local contexts. To learn more about how to fix these errors, check out these resources:',
+        {errorTypes: <b>Unbound local errors</b>}
+      ),
+      links: [
+        {
+          text: t('Fixing "unbound local" errors in Python'),
```

Comment:
here and in the description can we styled it as UnboundLocalError since that's how Python presents it